diam/sm/smparser/cea.go

```diff
@@ -57,6 +93,8 @@ func (cea *CEA) Parse(m *diam.Message, localRole Role) (err error) {
 		return err
 	}
 	cea.appID = app.ID()
+	cea.VendorID = 0
+	cea.ProductName = "totogi-ccab"
```

Comment:
Personally, I would like to have this repo be generic and not specific to totogi so that we can later create a PR to the upstream repo